code/modules/mob/living/living.dm

```diff
@@ -292,10 +292,16 @@
 		var/mob/M = AM
 
 		log_combat(src, M, "grabbed", addition="passive grab")
-		if(!supress_message && !(iscarbon(AM) && HAS_TRAIT(src, TRAIT_STRONG_GRABBER)))
-			M.visible_message("<span class='warning'>[src] grabs [M] [(zone_selected == "l_arm" || zone_selected == "r_arm")? "by their hands":"passively"]!</span>", \
-							"<span class='warning'>[src] grabs you [(zone_selected == "l_arm" || zone_selected == "r_arm")? "by your hands":"passively"]!</span>", null, null, src)
-			to_chat(src, "<span class='notice'>You grab [M] [(zone_selected == "l_arm" || zone_selected == "r_arm")? "by their hands":"passively"]!</span>")
+		if(!supress_message && !(iscarbon(AM) && HAS_TRAIT(src, TRAIT_STRONG_GRABBER))) //Everything in this if statement handles chat messages for grabbing
+			var/mob/living/L = M
+			if (L.getorgan(/obj/item/organ/tail) && zone_selected == BODY_ZONE_PRECISE_GROIN) //Does the target have a tail?
+				M.visible_message("<span class ='warning'>[src] grabs [L] by their [(L.getorgan(/obj/item/organ/tail/cat))? "cat":"lizard"] tail!</span>",\
+								"<span class='warning'> [src] grabs you by the tail! Ouch!</span>", null, null, src) //Message sent to area, Message sent to grabbee
+				to_chat(src, "<span class='notice'>You grab [L] by their [(L.getorgan(/obj/item/organ/tail/cat))? "cat":"lizard"] tail! Weirdo!</span>")  //Message sent to grabber
```

Comment:
```suggestion
			var/obj/organ = L.getorgan(/obj/item/organ/tail)
			if (organ && zone_selected == BODY_ZONE_PRECISE_GROIN) //Does the target have a tail?
				M.visible_message("<span class ='warning'>[src] grabs [L] by [L.p_their()] [organ]!</span>",\
								"<span class='warning'> [src] grabs you by the tail! Ouch!</span>", null, null, src) //Message sent to area, Message sent to grabbee
				to_chat(src, "<span class='notice'>You grab [L] by [L.p_their()] [organ]! Weirdo!</span>")  //Message sent to grabber
```

Reply:
needs testing to make sure this actually works since its a github webedit lol

Reply:
> 
> 
> needs testing to make sure this actually works since its a github webedit lol

Gives a shit ton of undefined vars.

Reply:
`grabs x by their cat tail` sounds bad. 
Well, worse